package.json

```diff
@@ -246,7 +247,8 @@
       "im.ltdev.cordova.UserAgent": {},
       "cordova-launch-review": {},
       "cordova-plugin-wkwebview-engine": {},
-      "cordova-plugin-advanced-http": {}
+      "cordova-plugin-advanced-http": {},
+      "cordova-plugin-add-swift-support": {}
```

Comment:
`cordova-plugin-add-swift-support` is added automatically when "prepare" mobile version. It's safe to remove that.